### PR TITLE
Crossfire LQ Formats

### PR DESCRIPTION
**Pull request by OptimusTi**
This update adds Crsf LQ formats to emulate TBS Hardware and BF style format. It also fixes character 234 of clarity_medium and impact_mini fonts.

### CODE REVIEW

File: tabs/osd.js [hunk 1]
```diff
@@ -1345,6 +1345,9 @@ OSD.constants = {
         },
         {
             name: 'osdGroupCRSF',
+            // enabled: function() {
```
Comment (DzikuVx) [inline tabs/osd.js:1348]: Please remove unused comments

File: tabs/osd.js [hunk 2]
```diff
@@ -1353,10 +1356,18 @@ OSD.constants = {
                     preview: FONT.symbol(SYM.RSSI) + '-100' + FONT.symbol(SYM.DBM)
                 },
                 {
-                    name: 'CRSF_LQ',
-                    id: 110,
-                    positionable: true,
-                    preview: '2:100%',
+                                    name: 'CRSF_LQ',
```
Comment (DzikuVx) [inline tabs/osd.js:1359]: formatting seems to be off here

**Review by DzikuVx — CHANGES_REQUESTED** [review]
a few nits a we should be ready to go

File: tabs/osd.js (hunk 2 above)
Comment (OptimusTi) [reply tabs/osd.js:1359]: Changes made on both PR. Thank you!

**Comment by DzikuVx** [comment]
@OptimusTi there seems to be a conflict with `234` character. Could you merge `master` into your branch and resolve the conflict?

**Review by DzikuVx — APPROVED** [review]

**Comment by OptimusTi** [comment]
> @OptimusTi there seems to be a conflict with `234` character. Could you merge `master` into your branch and resolve the conflict?

Hello Pawel, done. Thank you again.

**Pull request merged**